modules/javafx.base/src/main/java/javafx/util/converter/CurrencyStringConverter.java

```diff
@@ -32,8 +32,7 @@
 import javafx.util.StringConverter;
 
 /**
- * <p>{@link StringConverter} implementation for {@link Number} values
- * that represent currency.</p>
+ * A {@link StringConverter} implementation for {@link Number} values that represent currency. Instances of this class are immutable.
```

Comment:
Given that there are no setters or other mutator methods, the immutability seems obvious. I don't see a problem with stating it explicitly, though. It's up to you.

Reply:
True, but I just looked at [Border](https://openjfx.io/javadoc/14/javafx.graphics/javafx/scene/layout/Border.html) which states immutability, so I did the same. I don't mind changing it.

Reply:
I don't mind either way.

Reply:
I don't think it's bad to stating this explicitly here.

Reply:
Agreed.